### ISSUE TITLE
Get attribute name within normalizer

### ISSUE THREAD

**Issue by tanraya** [mdeering/attribute_normalizer#74]
I would like to create a normalizer with auto-truncation feature, so I need to know the current limit of a table field. To determine the length of the field I need to know the name of the attribute.

Here's what I want:

```ruby
config.normalizers[:auto_truncate] = lambda do |value, options|
  if value.is_a?(String)
    attribute_name = '' # ???
    length = columns_hash[attribute_name].limit
    text.mb_chars[0...length]
  else
    value
  end
end
```

I looked into the source and found nothing relative to the topic. What if I add :attribute_name key in options hash and send PR?

**Comment by mdeering** [comment]
Yes. I can see value in knowing the attribute and even having access to instance.
